src/Forms/book-form/BookForm.jsx

```diff
@@ -1,15 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Style from './BookForm.module.css';
 
-/* eslint-disable jsx-a11y/anchor-is-valid, react/jsx-curly-brace-presence */
-export default function BookForm() {
+/* eslint-disable jsx-a11y/anchor-is-valid, react/jsx-curly-brace-presence, react/prop-types */
+export default function BookForm({ addBookHandler }) {
+  const [form, setForm] = useState({ title: '', category: '' });
+  const handleInput = ($e) => {
+    setForm((pState) => ({ ...pState, [$e.target.name]: $e.target.value }));
+  };
+  const addNewBook = ($e) => {
+    $e.preventDefault();
+    addBookHandler(form);
+  };
+
   return (
     <div className={`${Style.FormContainer}`}>
       <h1 className={`${Style.FormTitle}`}>add new book</h1>
-      <form action="" method="post" className={`${Style.Form}`}>
-        <input type="text" placeholder="Book Title" className={`${Style.FormInput}`} />
-        <select className={`${Style.FormCategory}`} name="Catagory" placeholder="Category" id="">
-          <option value="catagory1">Cat 1</option>
+      <form action="" method="post" onSubmit={addNewBook} className={`${Style.Form}`}>
+        <input type="text" name="title" placeholder="Book Title" onInput={handleInput} className={`${Style.FormInput}`} />
+        <select className={`${Style.FormCategory}`} onChange={handleInput} value={form.category} name="category" placeholder="Category" id="">
+          <option value="catagory 1">Cat 1</option>
```

Comment:
- [ ] As pointed out by the previous reviewer, your add book form is missing the `Author field`. Your Add book form is meant to have 4 inputs, `Title, Author, and Category`. I'll suggest you add the Author's input to the form. :+1: :smile: 

- - Screenshot of the required inputs:
![Screenshot from 2023-07-29 10-01-02](https://github.com/habitoz/awesome-book-react/assets/107632780/ba5b9ff2-5a5c-4d5f-b35f-f2b417c13d44)